.github/workflows/promote.yaml

```diff
@@ -30,11 +30,10 @@ jobs:
           destination=$(echo "$channel_promotion" | sed 's/.*->\s*//')
           echo "destination-channel=$destination" >> $GITHUB_OUTPUT
           echo "origin-channel=$origin" >> $GITHUB_OUTPUT
-      - name: Release charm to channel
-        uses: canonical/charming-actions/release-charm@2.6.2
+      - name: Promote charm to channel
+        uses: canonical/charming-actions/promote-charm@2.6.3
         with:
           credentials: ${{ secrets.CHARMHUB_TOKEN }}
-          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this correct to remove the github-token?

Reply:
I believe so; this change was in @jneo8's original PR: https://github.com/canonical/charm-juju-backup-all/pull/58/files